packages/rs-drive-abci/src/error/execution.rs

```diff
@@ -21,6 +21,10 @@ pub enum ExecutionError {
     #[error("drive incoherence error: {0}")]
     DriveIncoherence(&'static str),
 
+    /// Error
+    #[error("upgrade incoherence error: {0}")]
+    UpgradeIncoherence(&'static str),
```

Comment:
ProtocolUpgradeIncoherence ?

Reply:
Is there any other type of upgrade?

Reply:
I can't say right now but better to have such things implicit, right? We use term Protocol Upgrade

Reply:
Fine